src/render/PointArray.cpp

```diff
@@ -154,8 +154,8 @@ static OctreeNode* makeTree(int depth, size_t* inds,
         return node;
     }
     // Partition points into the 8 child nodes
-    size_t* childRanges[9];
-    multi_partition(beginPtr, endPtr, OctreeChildIdx(P, center), childRanges+1, 8);
+    std::vector<size_t *> childRanges(9, NULL);
```

Comment:
I assume this is about initialization, can we get away with `size_t* childRanges[9] = {0};` instead?  Seems a bit unfortunate to hit the allocator for this small fixed size array.